tests/e2e/configuration/library-mode/lightspeed-stack-legacy.yaml

```
@@ -0,0 +1,48 @@
+    tool:
+      sources:
+        - e2e-test-docs
+
+shields:
+  - name: pii-redaction
+    provider_id: redaction
+    config:
+      rules:
+        - pattern: '\d+'
+          replacement: '[NUM]'
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)